src/Servers/IIS/AspNetCoreModuleV2/IISLib/acache.cpp

```diff
@@ -429,7 +429,6 @@ ALLOC_CACHE_HANDLER::IsPageheapEnabled(
 
     if ( fLockedHeap )
     {
-        fLockedHeap = FALSE;
```

Comment:
Is there an analyzer warning about setting a local without ever reading it again? Not sure how I feel about that in the general case 😆
For this file it's fine since we don't touch it.

Reply:
Haha, I hit that while trying out clang-tidy locally and fixed it in this super obvious case.